lib/index.d.ts

```diff
@@ -30,7 +35,7 @@ declare namespace stringify {
          * can refer to nested properties of the input JSON
          * see the "header" option on how to print columns names on the first line
          */
-        columns?: string[] | PlainObject<string>
+        columns?: string[] | PlainObject<string> | ColumnOptions[]
```

Comment:
And then in here you'd have the plural: an array of column options.

```suggestion
        columns?: string[] | PlainObject<string> | ColumnOption[]
```